server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -205,7 +205,10 @@ public enum MetadataVersion {
     IBP_3_8_IV0(20, "3.8", "IV0", true),
 
     // Introduce version 1 of the GroupVersion feature (KIP-848).
-    IBP_4_0_IVO(21, "4.0", "IV0", false);
+    IBP_4_0_IVO(21, "4.0", "IV0", false),
```

Comment:
Not introduced in this change, but looks like IBP_4_0_IVO uses letter "O" at the end which is different from other versions that use number "0".

Reply:
Hehe. This will be fixed when I fix conflicts. But good to keep in mind 👍